portal-api/api-tests/fixtures/constants.js

```diff
@@ -0,0 +1,16 @@
+const DB_COLLECTIONS = {
```

Comment:
Discussed: `portal-api` should not be able to interact directly with the database except in the case of running these tests. Adding this to `src/constants` would imply that `portal-api` can interact directly with the database outside of the tests which is something we want to avoid (in this ticket anyway)